### PR TITLE
Refactor Landing Page components into smaller ones

### PR DESCRIPTION
**Pull request by KarinaDavtyan**
`Headline` and `Info` related components are in the folders now

### CODE REVIEW

File: apps/landingPage/src/components/Headline/Subtitle.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+|};
+
+export default function Subtitle(align: Props) {
+  return (
+    <Container>
+      <HeaderAndDesignWrapper>
+        <BackgroundGradientAccent>
+          <GreenRectangle />
+        </BackgroundGradientAccent>
+        <Text align={!align} type="title2" text={bodyText} />
```
Comment (jvaclavik) [inline apps/landingPage/src/components/Headline/Subtitle.js]: Maybe we should keep it similar with Text from margarita and have text as a children

Comment (KarinaDavtyan) [reply apps/landingPage/src/components/Headline/Subtitle.js]: Changed `prop` to children and also renamed component to the `TextHeadline`

Comment (tbergq) [reply apps/landingPage/src/components/Headline/Subtitle.js]: this looks weird `align={!align}`

Comment (KarinaDavtyan) [reply apps/landingPage/src/components/Headline/Subtitle.js]: is `align={align}` less weird? as I just changed it to that

Comment (tbergq) [reply apps/landingPage/src/components/Headline/Subtitle.js]: Yes, it is better 😄

File: apps/landingPage/src/components/Info/Features.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import React from 'react';
+import { Heading } from '@kiwicom/orbit-components/lib/';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import styled from 'styled-components';
+
+import { BREAKPOINTS } from '../../mediaQueriesConfig';
+
+const features = ['Search', 'Booking', 'MMB', 'Payments'];
+
+const renderCards = (features: Array<string>) => {
```
Comment (tbergq) [inline apps/landingPage/src/components/Info/Features.js]: could you use `$ReadOnlyArray<string>` instead?

Comment (KarinaDavtyan) [reply apps/landingPage/src/components/Info/Features.js]: Sure! changed now

File: apps/landingPage/src/components/Info/Features.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  ));
+};
+
+export default function Features() {
+  return (
+    <Container>
+      <Title>
+        <Heading>Features</Heading>
+      </Title>
+      <CardListWrapper>{renderCards(features)}</CardListWrapper>
```
Comment (tbergq) [inline apps/landingPage/src/components/Info/Features.js]: Also, I don't understand why renderCards is a function. It looks and acts like a component. 
What about 
```
const RenderCards = ({features}: Props) => {
return features.map(feature => (
    <CardWrapper key={feature}>
      <Heading type="title2">{feature}</Heading>
    </CardWrapper>
  ));
}

...
 <CardListWrapper><RenderCards features={features} /></CardListWrapper>
```

Comment (KarinaDavtyan) [reply apps/landingPage/src/components/Info/Features.js]: Ok! changed to the component

**Comment by KarinaDavtyan** [comment]
> Looks much better now with smaller components 🙂

Yes, it is! 🙂

**Pull request merged**